src/components/10-atoms/button/index.js

```diff
@@ -81,11 +81,11 @@ class AXAButton extends LitElement {
     // If someone fires a click on the button and its type is submit then trigger fake button
     // press
     this.onclick = e => {
-      this.handleClick(e);
+      this.handleClick(e, true);
     };
   }
 
-  handleClick = e => {
+  handleClick = (e, ignoreOnClick = false) => {
```

Comment:
I was a bit confused about that attribute `ignoreOnClick`. A function "handleClick" with an attribute "ignoreOnClick" is worthless?! Perhaps a more accurate name would be better to read.

Reply:
@danyball unfortunately is necessary. the comment:    `// If someone fires a click on the button and its type is submit then trigger fake button` should explain why